res/sections/20-WorkingEnvironment.tex

```diff
@@ -0,0 +1,267 @@
+\chapter{Ambiente di Lavoro}
+
+Per far funzionare \LaTeX{} sul vostro computer dovete scaricare anche un
```

Comment:
Mi Sembra brutto l'anche perchè non mi pare che abbiamo parlato in precedenza di cose da scaricare